classes/ActionScheduler_QueueRunner.php

```diff
@@ -53,6 +53,10 @@ public function run() {
 		return $count;
 	}
 
+	public function get_concurrent_batch_count() {
```

Comment:
Would it make sense to move this to the `ActionScheduler_Abstract_QueueRunner` class? I think other queue runners could benefit from it.

Reply:
@JPry yep for sure. I'll also rename it to `get_allowed_concurrent_batches()` to avoid any ambiguity - the current `get_concurrent_batch_count()` could be confused with returning the number of batches in-progress rather than the maximum number allowed to run concurrently.